PhysicsTools/PatAlgos/python/slimming/lowPtGsfLinks_cfi.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+lowPtGsfLinks = cms.EDProducer(
+    'LowPtGSFToPackedCandidateLinker',
```

Comment:
Do not write the producer in a new line, so that to separate it visually from the other parameter